CRM/Contact/BAO/Group.php

```diff
@@ -439,8 +451,10 @@ public static function create(&$params) {
 
       if (!empty($params['parents'])) {
         foreach ($params['parents'] as $parentId => $dnc) {
-          if ($parentId && !CRM_Contact_BAO_GroupNesting::isParentChild($parentId, $group->id)) {
-            CRM_Contact_BAO_GroupNesting::add($parentId, $group->id);
+          if (CRM_Utils_Type::validate($parentId, 'Integer')) {
```

Comment:
Well let's merge it as is if it passes - I think I fixed that function so it won't matter but it won't hurt